app/src/main/java/mod/elfilibustero/sketch/lib/handle/BuiltInUtilClassHandler.java

```diff
@@ -0,0 +1,90 @@
+package mod.elfilibustero.sketch.lib.handle;
+
+import a.a.a.eC;
+import a.a.a.hC;
+
+import com.besome.sketch.beans.ComponentBean;
+
+public class BuiltInUtilClassHandler {
+
+    /**
+     * Check if any SketchwareUtil block is used in the project.
+     */
+    public static boolean isSketchwareUtilBlockUsed(hC projectFileManager, eC projectDataManager) {
+        for (var activity : projectFileManager.b()) {
+            for (var entry : projectDataManager.b(activity.getJavaName()).entrySet()) {
+                for (var blockBean : entry.getValue()) {
+                    if (isSketchwareUtilPresent(blockBean.opCode)) {
+                        return true;
+                    }
+                }
+            }
+        }
+        
+        return false;
+    }
+
+    /**
+     * Check if specific SketchwareUtil block is present.
+     */
+    public static boolean isSketchwareUtilPresent(String opCode) {
+        String[] blocks = {
```

Comment:
this would create util class even if you use some custom block with this name, i think

Reply:
wouldn't that be the fault of the Custom Block creator since they used an already existing Block opcode

Reply:
i dont think they'd know that much about internals tbh

Reply:
since that'd also override built-in Blocks' code and so on, this is nothing to worry about, it's just more damage from doing what shouldn't be done unless done carefully